setup.py

```diff
@@ -65,7 +68,7 @@
         'Intended Audience :: Developers',
         'Intended Audience :: Education',
         'Intended Audience :: Science/Research',
-        'License :: OSI Approved :: MIT License',
+        'License :: OSI Approved :: Apache License 2.0',
```

Comment:
I am putting this in a license kwarg, but using `Apache Software License` here as per https://pypi.org/classifiers/, since this might not be recognizable by automated tools. We can have a more specific license here once https://github.com/pypa/pypi-support/issues/142 is addressed.